### ISSUE TITLE
UIUX: Improvement of balance display

### ISSUE THREAD

**Issue by moneymanolis** [cryptoadvance/specter-desktop#1627]
Once #1580 is merged:

![image](https://user-images.githubusercontent.com/60378539/157185870-e995e0d9-10c3-4197-837c-4a09a5af081b.png)
It would be even nicer if the amounts would be aligned such that the decimal separator is at the same x position.   And for completeness the unit  (tBTC) after each amount would be good too.

_Originally posted by @relativisticelectron in https://github.com/cryptoadvance/specter-desktop/pull/1580#discussion_r821382127_

**Comment by moneymanolis** [comment]
Quoting relevant comments from @cypherhoodlum and his PR (already a lot of CSS etc. groundwork in there) here as well:

https://github.com/cryptoadvance/specter-desktop/pull/1580#issuecomment-1075713089
https://github.com/cryptoadvance/specter-desktop/pull/1580#issuecomment-1075749817
https://github.com/moneymanolis/specter-desktop/pull/1

**Comment by k9ert** [comment]
The last two boxes could be ticked by @cypherhoodlum PR here: https://github.com/moneymanolis/specter-desktop/pull/1

**Comment by moneymanolis** [comment]
Yes, the main CSS approach is already in there.
But: x position is not fixed by it (only looks like it since the amounts have the same length)
And also, I am not completely sure whether the approach is the best regarding readability. Since this is a very important element, I think it is worth giving it a bit more thought. 
Tagging @kdmukai here as well, the master of seed signer balance displaying.

**Comment by cypherhoodlum** [comment]
It seems this issue requires the #1580 to be merged into the master branch first:
<img src="https://user-images.githubusercontent.com/82510823/160277064-f4fd25d2-4566-43e3-9f4c-dfe04f1cd812.png" width="600" />
Or is there something I'm missing @k9ert @moneymanolis?

**Comment by moneymanolis** [comment]
@cypherhoodlum #1580 is close to be merged. Finally, the CI worked again.

**Comment by cypherhoodlum** [comment]
Good to hear, seemed like some nasty problems. I will finish styling the balance display once #1580 is fully merged.

**Comment by moneymanolis** [comment]
fyi @cypherhoodlum: #1580 has been merged

**Comment by cypherhoodlum** [comment]
Sorry, it's been a while. I'm starting to think the fixed x position based on the decimal point is too ambitious given all that's happening in the balances display. Here's how my proposal looks right now:

<img src="https://user-images.githubusercontent.com/82510823/164519746-9f2b6259-35e4-4aa2-a0b8-34a4aa714567.png" width="600" />

<img src="https://user-images.githubusercontent.com/82510823/164519413-0ac25e43-e54d-4d7a-a64d-b3f57e918b77.png" width="600" />

<img src="https://user-images.githubusercontent.com/82510823/164519616-84ebda33-23c5-4633-85b3-ae98ea66b657.png" width="600" />

There are no additional hovering effect. It's not perfect design but I think it's still an improvement. Any thoughts?

**Comment by relativisticelectron** [comment]
> Sorry, it's been a while. I'm starting to think the fixed x position based on the decimal point is too ambitious given all that's happening in the balances display.

x alignment according to the decimal separator would improve readability a lot.   Maybe the decimal numbers can be split at the decimal separator to get this x-alignment:   (right aligned)  "5,"      and (left aligned) "12345678  tBTC"

Can you reduce the empty y distances/margins?  The Balance display should not take too much y space.

**Comment by cypherhoodlum** [comment]
I reduced the y margins and attempted to fix the decimal separator in the x axis but this needs some more testing. Preferably by multiple people. I don't trust my numbers. Here's how it looks right now:
<img src="https://user-images.githubusercontent.com/82510823/164764344-3e3c7df7-31e2-4f0f-a1e4-3bd523bb3b65.png" width="600" />

Should I make a PR?

**Comment by relativisticelectron** [comment]
This looks a lot better. It would be great to reduce the x and y spaces even more to make it more compact.   Where would the fiat amount appear?

However when I checked out your [branch](https://github.com/cypherhoodlum/specter-desktop/tree/improvement_of_balance_display)     it looks like:
![image](https://user-images.githubusercontent.com/60378539/164887356-ef26e961-3109-4937-99a4-b5739e7adcf6.png)
Are the newest changes maybe not on the branch yet?

**Comment by PaarthAgarwal** [comment]
While we are at it I believe this issue has been completed as well

**Comment by moneymanolis** [comment]
Actually, not quite, we removed any special styling of balances in the uiux revamp. But I am still closing this one, since it makes more sense to open a new one with the new UI.
